config/config_mongodb_continual.ini

```diff
@@ -3,7 +3,7 @@ interface=MongoDB
 
 [mongodb]
 host=localhost
-db=Scenes
+db=UTest
```

Comment:
this might break the tutorials online

Reply:
Should we use a diiferent config file or use the Scenes database name?

Reply:
different config files (e.g. data_loader_uTest);  I am preparing a new set of images for this; Fee free to modify that config file to suite your needs.